src/org/benf/cfr/reader/entities/classfilehelpers/ClassFileDumperAnnotation.java

```diff
@@ -43,7 +43,7 @@ public Dumper dump(ClassFile classFile, InnerClassDumpType innerClass, Dumper d)
         boolean first = true;
         dumpAnnotations(classFile, d);
         dumpHeader(classFile, innerClass, d);
-        d.print("{").newln();
+        d.separator("{").newln();
```

Comment:
Should the closing `}` in line 67 be a separator as well?

Same for ClassFileDumperAnonymousInner.java, ClassFileDumperEnum.java and ClassFileDumperInterface.java.